src/modules/explorer/pages/Proposals/index.tsx

```diff
@@ -233,6 +253,18 @@ export const Proposals: React.FC = () => {
             </TabsContainer>
           </Grid>
 
+          <FiltersContainer
+            onClick={() => setOpenFiltersDialog(true)}
+            xs={isMobileSmall ? 12 : 2}
+            item
+            container
+            direction="row"
+            alignItems="center"
+          >
+            <FilterAltIcon style={{ color: theme.palette.secondary.main }} fontSize="small" />
+            <Typography color="secondary">Filter & Sort</Typography>
```

Comment:
Not working for me the first time I select `Active` or `Closed` it just shows No Proposal found. Then second time it works

Reply:
Yes, All good now.